openslides_backend/shared/otel.py

```diff
@@ -0,0 +1,56 @@
+import os
```

Comment:
Since this file doubles so much with the datastore one, you could just import the functionality from the datastore here, since it is embedded as a python package. You could still keep the file so it is easily extendable for the backend if needed (and for backend specific functionality, like the requests instructor).